docs/mindsdb-docs/docs/sql/create/predictor.md

```diff
@@ -164,7 +164,7 @@ USING
     model.args='{"submodels":[
```

Comment:
We can specify json arguments in both ways: as strings and as json itself:
```
arg='{"x": 1}'
or
arg={"x": 1}
```
I like the second option much more, it is more obvious. Let delete `'` around json in example

Reply:
Changed.

Reply:
Also, I will make this change to all other queries in the docs to keep them uniform.